content/rule_book/quiz_process.md

```diff
@@ -1,106 +1,173 @@
-- If questions 1-15 are erred, the next question will be a bonus question.
-- If questions 16-20 are erred, 16A-20A will be a bonus question.
-- All bonus questions throughout the whole quiz are 10 points.
+- Additional units of 3 overtime questions each will be used until the tie is broken
+- 2-team rules will apply in overtime if only 2 teams are involved
+- All the numeric questions will be of a different question type
 
-### Overtime
+## Question Type Distribution
 
-- Additional units of three overtime questions each will be used until the tie is broken.
-- Two-team rules will apply in overtime if only two teams are involved.
-- All the question numbers will be of a different question type.
+In the below table, any minimum or maximum variation between material seasons is indicated in a `Narrative/Epistle` format.
 
-## Platform Protocol
+| Type Group      | Minimum | Maximum | Question Types       |
+|-----------------|--------:|--------:|----------------------|
+| Interrogative   |     8/9 |   14/16 | INT                  |
+| Multiple Answer |       1 |       2 | MA                   |
+| Reference       |       3 |     6/7 | CR, CVR, MACR, MACVR |
+| Quote           |     2/3 |     3/4 | Q, Q2V               |
+| Finish          |       3 |       4 | FT, FTN, FTV, F2V    |
+| Situation       |     2/0 |     4/0 | SIT                  |
+
+For a given Question Type:
+
+- The minimum number of questions must be asked in question numerics 1-20, thus excluding any alphanumeric question numbers
+- The maximum number of questions must not be exceeded in all "non-overtime" questions. this includes both numeric-only and alphanumeric question numbers
 
-    Set the readiness bonus to 20.
+For 15-question quizzes, prepare quizzes as normal for 20-question quizzes, but the quizmaster should conclude the quiz when appropriate for the rule of a 15-question quiz.
+
+## Platform Protocol
 
 - Quizzers will be seated facing the audience and quizmaster.
 - The feet of the quizzers are to be on the floor while jumping.
 - A line-up card showing the seating order of each quizzer should be given to the quizmaster or scorekeeper before the start of each quiz.
 - No verbal or non-verbal communication is permitted between quizzers, coaches, or the audience from the time the quizmaster introduces the question type until points have been awarded.
-- Only a coach from a team presently participating in the quiz, or the meet director may confer with the officials.
+- Only a coach from a team presently participating in the quiz or the meet director may confer with the officials.
 - Coaches may have quiz material(s) on the platform during a timeout or prior to a quiz.
 - Quizzers will not touch the officials' table.
 - Quiz rooms will use jump measuring equipment to prevent ties.
 - Push-button quizzing can be provided if needed by physically disabled quizzers.
 
 ## Jumping
 
+- Quizzers have 5 seconds in which to jump after the question is completed.
```

Comment:
"...after the quizmaster finishes reading the question" - better?

Reply:
I agree. f13388d